### PR TITLE
update/SOF 7453

### PR DESCRIPTION
**Pull request by VsevolodX**
- **update: add function to create empty material**
- **update: add tests for empty material**
- **update: test to sync cell with lattice**

### CODE REVIEW

File: src/py/mat3ra/made/tools/modify.py [hunk 1]
```diff
@@ -40,32 +40,33 @@ def filter_by_label(material: Material, label: Union[int, str]) -> Material:
...
     elif z_level == "center":
         material = translate_by_vector(material, vector=[0, 0, (1 - min_z - max_z) / 2])
     return material
 
 
 def translate_by_vector(
-    material: Material,
-    vector: Optional[List[float]] = None,
-    use_cartesian_coordinates: bool = False,
+        material: Material,
```
Comment (timurbazhirov) [inline src/py/mat3ra/made/tools/modify.py:67]: Indentation needs to be fixed

File: tests/py/unit/test_material.py [hunk 2]
```diff
@@ -25,4 +25,68 @@ def test_basis_to_json():
...
+    assert material.lattice.beta == 90.0
+    assert material.lattice.gamma == 90.0
+    assert material.name == "New Material"
+
+    # Test custom parameters
+    material = Material.create_empty(
+        a=2.0, b=3.0, c=4.0, alpha=80.0, beta=85.0, gamma=95.0, lattice_type="TRI", name="Custom Empty"
+    )
+    assert material.basis.elements.values == []
+    assert material.basis.coordinates.values == []
```
Comment (timurbazhirov) [inline tests/py/unit/test_material.py:62]: Test empty should only check the emptiness.

The actual parameter values should be in test_create.

On the material level we should just check lattice and basis, name - top-level params

For lattice/basis we should have separate tests (create, etc) that check their internal parameters

File: src/py/mat3ra/made/material.py [hunk 3]
```diff
@@ -112,3 +112,58 @@ def add_atom(self, element: str, coordinate: List[float], use_cartesian_coordina
...
+            "gamma": gamma,
+            "units": {
+                "length": Units.angstrom,
+                "angle": Units.degree,
+            },
+        }
+
+        config = {"name": name, "basis": basis_config, "lattice": lattice_config}
+
+        return cls(config)
```
Comment (timurbazhirov) [inline src/py/mat3ra/made/material.py:169]: Too much code - should just be `self.create(config_with_empty_basis)

File: src/py/mat3ra/made/material.py [hunk 4]
```diff
@@ -112,3 +112,58 @@ def add_atom(self, element: str, coordinate: List[float], use_cartesian_coordina
...
+            "alpha": alpha,
+            "beta": beta,
+            "gamma": gamma,
+            "units": {
+                "length": Units.angstrom,
+                "angle": Units.degree,
+            },
+        }
+
+        config = {"name": name, "basis": basis_config, "lattice": lattice_config}
```
Comment (timurbazhirov) [inline src/py/mat3ra/made/material.py:167]: This is code duplicate - needs to be handled at the lattice level or through self.create

File: src/py/mat3ra/made/material.py [hunk 5]
```diff
@@ -112,3 +112,52 @@ def add_atom(self, element: str, coordinate: List[float], use_cartesian_coordina
         new_basis = self.basis.copy()
         new_basis.add_atom(element, coordinate, use_cartesian_coordinates)
         self.basis = new_basis
+
+    @classmethod
+    def create_empty(
```
Comment (timurbazhirov) [inline src/py/mat3ra/made/material.py:117]: Let's remove this

**Review by timurbazhirov — APPROVED** [review]
